hack/lib/build/version.sh

```diff
@@ -88,10 +89,11 @@ function os::build::version::kubernetes_vars() {
 	# Try to match the "git describe" output to a regex to try to extract
 	# the "major" and "minor" versions and whether this is the exact tagged
 	# version or whether the tree is between two tagged versions.
-	if [[ "${KUBE_GIT_VERSION}" =~ ^v([0-9]+)\.([0-9]+)(\.[0-9]+)*([-].*)?$ ]]; then
+	if [[ "${KUBE_GIT_VERSION}" =~ ^v([0-9]+)\.([0-9]+)\.([0-9]+)(\.[0-9]+)*([-].*)?$ ]]; then
```

Comment:
Reverted in https://github.com/openshift/origin/pull/14967 since we don't really need it for kube, just OS.